docker-compose.yml

```diff
@@ -67,6 +67,7 @@ services:
     working_dir: /offen/auditorium
     environment:
       LOCALE: ${LOCALE:-en}
+      SCRIPT_INTEGRITY_HASH: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
```

Comment:
Would it make sense to use sth like `THIS_IS_A_PLACEHOLDER_VALUE` so it's a bit more obvious this won't really work?

Reply:
I think that's okay. 

It's probably okay that during development there is no real chance to test the integrity function since you only need it in production anyway. So we can clearly mark it as placeholder.